aerogear/mobile-developer-console
# Aerogear 9500

## Motivation
Wrong description shown for services.
Issue : https://issues.jboss.org/browse/AEROGEAR-9500

## Why
Services description data fields were referencing the service display name.

## How
Change to UnboundServiceRow and BoundServiceRow render methods.

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Create an app.
2. Go to the app page.
3. Click the Services tab
4. In the list unbound services you will see the services name has a different title and subtitle.
5. Add service to bound services
6. In the list bound services you will see the service name has a different title and subtitle.

## Checklist:
- [ ] Services show different title and subtitle
